build/npm/waypoint-hoc.js

```diff
@@ -218,6 +218,11 @@ var waypoint = function waypoint(Component) {
           });
         }
       }
+
+      /**
+       * @return {Object} `height` and `top` of scrollable ancestor Rect.
+       */
+
     }, {
       key: '_scrollableAncestorHeightTop',
       value: function _scrollableAncestorHeightTop() {
```

Comment:
We need build folder on repo for `npm install https://github.com/brigade/react-waypoint.git#master` cases.